content/applications/inventory_and_mrp/purchase/products/reordering.rst

```diff
@@ -1,96 +1,82 @@
------------------
+   .. example::
+      If :guilabel:`Max Quantity` is set to `100` but :guilabel:`Multiple Quantity` is set to order
+      the product in batches of `200`, a purchase order will be created for 200 units of the
+      product.
 
-You can set a **minimum quantity** your stock should always have. Once set, if your stock goes below
-the minimum quantity, and if you selected the Buy route, a request for quotation is automatically
-generated to reach that minimum quantity, plus any additional quantity needed to fill in a sales
-order for example.
+- :guilabel:`UoM` specifies the unit of measurement by which the quantity will be ordered. For
+  discrete products, this should be set to `Units`. However, it can also be set to units of
+  measurement like `Volume` or `Weight` for non-discrete products like water or bricks.
 
-If you set a **maximum quantity**, every time the product has to be replenished, enough products are
-reordered to reach the maximum quantity.
+.. image:: reordering/reordering-rule-configuration.png
+   :align: center
+   :alt: Configure the reordering rule in Odoo.
 
-If you want to order only the exact quantity needed to fill in a sales order for example, set both
-both the minimum and maximum quantity to **zero**. The quantity mentioned in the sales order is then
-used by the reordering rule.
+Triggering reordering rules using the scheduler
+===============================================
 
-You can also add a **quantity multiple** to only order products in batches of a certain quantity.
-Click on the optional columns drop-down menu, and select *Multiple Quantity* to show the column.
-Bear in mind that you might go over the maximum quantity you set if your rule includes a quantity
-multiple, as Odoo orders enough products to reach the maximum quantity and respect the set quantity
-multiple.
+Reordering rules will be automatically triggered by the scheduler, which runs once a day by
+default. To trigger reordering rules manually, navigate to
+:menuselection:`Inventory --> Operations --> Run Scheduler`.
 
-.. image:: reordering/reordering-create-rule.png
-   :align: center
-   :alt: Create a reordering rule in Odoo
+.. note::
+   Manually triggering reordering rules will also trigger any other scheduled actions.
 
-.. note:: If you selected multiple routes for the same product under its Inventory tab, make sure to
-   select your *Preferred Route* on your reordering rule by clicking on the optional columns
-   drop-down menu, adding the *Preferred Route* column, and selecting the right route.
+Managing reordering rules
+=========================
 
-.. Add link when inventory doc on replenishment is updated for v14. .. seealso:: - :doc:`../../..inventory/xxx/xxx`
+To manage the reordering rules for a single product, go to the product page and select
+:guilabel:`Reordering Rules`. To manage all reordering rules for every product, go to
+:menuselection:`Inventory --> Configuration --> Reordering Rules`.
```

Comment:
Expanded on this some more for more context and so it closed nicely on familiar features to Odoo users.

```suggestion
To manage the reordering rules for a single product, navigate to that product page's form and select
the :guilabel:`Reordering Rules` tab at the top of the form.

To manage all reordering rules for every product, go to :menuselection:`Inventory --> Configuration
--> Reordering Rules`. From this dashboard, typical bulk actions in Odoo can be performed such as
exporting data or archiving rules that are no longer needed. As well, the :guilabel:`Filters`,
:guilabel:`Group By` or triple-dotted menu on the form are available to search for and/or organize
the reordering rules as desired.
```